src/gl/m420-to-rgb-gl.cpp

```diff
@@ -0,0 +1,216 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#include <librealsense2/hpp/rs_sensor.hpp>
+#include <librealsense2/hpp/rs_processing.hpp>
+#include <librealsense2-gl/rs_processing_gl.hpp>
+
+#include "../proc/synthetic-stream.h"
+#include "m420-to-rgb-gl.h"
+#include "../option.h"
+
+#ifndef NOMINMAX
+#define NOMINMAX
+#endif // NOMINMAX
+
+#include <glad/glad.h>
+
+#include <iostream>
+
+#include <chrono>
+#include <strstream>
+
+#include "synthetic-stream-gl.h"
+
+static const char* fragment_shader_text =
```

Comment:
Please add some explanation for this, or remember to do so on the next PR

Reply:
I ll do so in the PR in which this code will be updated